Resources.html

```diff
@@ -81,25 +101,65 @@
+            <a href="https://cyberdefenders.org/" target="_blank"><b>Cyber Defenders</b></a><p>A training platform with a focus on defending networks from malicious attacks.</p> 
+            <a href="https://blueteamlabs.online/" target="_blank"><b>Blue Team Labs</b></a><p>A similar platform to train in defensive techniques related to security investigations.</p>
+          </div>
+          <div class="row"> 
+            <div class="div-highlight">
+            <h3>Where to learn Linux</h3>
+            </div>
+            <a href="https://linuxjourney.com/" target="_blank"><b>Linux Journey</b></a><p>An easy-to-follow guide on everything you should know about Linux that comes with helpful examples to reinforce your knowledge.</p>
+            <a href="https://www.linuxcommand.org/" target="_blank"><b>Linux Command</b></a><p>Focuses mainly on terminal commands, shell scripts, and learning to get a handle on optimizing your experience using Linux.</p>
+          </div>
+          <div class="row">
+            <div class="div-highlight">
+            <h3>Guides</h3>
+            </div>
+            <i> coming soon </i>
+          </div>
+      </div>
+      <div class="col-sm-6 col-md-6 col-lg-6 text-left">
+          <div class="row">
+            <div class="div-highlight">
+            <h3>Stay in the know</h3>
+            </div>
+            <a href="https://owasp.org/Top10/" target="_blank"><b>OWASP Top Ten</b></a><p>The top 10 categories of vulnerabilities found on websites. This is an updated document used by organizations to assess risks and implement changes to prevent these types of vulnerabilities from being found and exploited.</p> 
+            <a href="https://krebsonsecurity.com/" target="_blank"><b>Krebs on Security</b></a><p>A popular security focused blog by Brian Krebs. Frequently updated with detailed news of hacks and information disclosures and leaks.</p>
+            <a href="https://blog.malwarebytes.com/" target="_blank"><b>Malwarebytes Blog</b></a><p>Good general information blog about all things related to cybersecurity, reports, privacy, and news on software patches.</p>
+            <a href="https://www.bleepingcomputer.com/" target="_blank"><b>Bleeping Computer</b></a><p>General computing blog that includes cybersecurity, tutorials, deals on paid training platforms and software, and forums.</p>
+          </div>
+          <div class="row">
+            <div class="div-highlight">
+            <h3>Recommended reading</h3>
+            </div>
+            <a href="https://dfirmadness.com/getting-into-infosec/the-five-pillars/" target="_blank"><b>The Five Pillars</b></a><p>A detailed layout of the infosec field along with myriad resources of learning material and clear explanations of the different roles available in the industry.</p> 
```

Comment:
Slightly confused about the following bit:
> along with myriad resources of learning material...
Just the wording feels weird and slightly confusing.
Did you mean to say "along with a myriad of resources, learning material..."?